lib/team.rb

```diff
@@ -0,0 +1,27 @@
+class Team
+  attr_reader   :country,
+                :players
+  attr_accessor :eliminated
```

Comment:
👍

Since you're providing access to the `@eliminated` instance variable through the method `eliminated?` consider switching this to an `attr_writer`.